es6-promises/index.js

```diff
@@ -0,0 +1,10 @@
+const takeAChance = require('./take-a-chance');
+const promiseObject = takeAChance('Andre');
+
+promiseObject.then(value => {
+  console.log(value);
+});
+
+promiseObject.catch(error => {
+  console.error(error);
+});
```

Comment:
Make sure only the message shows, not the whole `Error` object.

---

![image](https://user-images.githubusercontent.com/16143771/178063593-9e880e3c-4d38-45ce-b0e3-de13ceb2ca6e.png)